docs/guides/info_http_api.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: "cloudamqp"
+page_title: "HTTP API"
```

Comment:
to make it more clear as just "HTTP API" is very generic term (needs updating in more places)

```suggestion
page_title: "Message Broker HTTP API"
```

Reply:
Good point.